collector/src/main/java/io/prometheus/jmx/JmxCollector.java

```diff
@@ -60,18 +62,20 @@ private JmxCollector(Map<String, Object> config) throws MalformedObjectNameExcep
             config = new HashMap<String, Object>();
         }
 
-        if (config.containsKey("hostPort")) {
-          hostPort = (String)config.get("hostPort");
-        } else {
-          // Default to local JVM.
```

Comment:
I think we should keep the else and this comment so that it's explicit what the default is.